appveyor.yml

```diff
@@ -79,6 +81,12 @@
         appveyor_repo_tag: true       # deploy on tag push only
 
 -
+  branches:
+    except:
+      - release
+
+  skip_non_tags: false
```

Comment:
I would think you only need here if the configs cascade which I don't believe they do. Try removing this and see if AppVeyor still builds the branch.